release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3627,6 +3627,8 @@ metadata:
 
 * Previously, the etcd Operator checked the health of etcd members in serial with an all-member timeout that matched the single-member timeout. That allowed one slow member check to consume the entire timeout, and cause later member checks to fail with the error `deadline-exceeded`, regardless of the health of that later member. Now, etcd checks the health of members in parallel so the health and speed of one member's check doesn't affect the other members' checks. (link:https://issues.redhat.com/browse/OCPBUGS-36489[*OCPBUGS-36489*])
 
+* Previously, you could not change the snapshot limits for the {vmw-first} Container Storage Interface (CSI) driver because of an error with the Cluster Storage Operator. With this release, you can now change the snapshot limits as expected. For more information about changing the snapshot limits, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.html#vsphere-change-max-snapshot_persistent-storage-csi-vsphere[Changing the maximum number of snapshots for vSphere] (link:https://issues.redhat.com/browse/OCPBUGS-36969[*OCPBUGS-36969*])
```

Comment:
```suggestion
* Previously, you could not change the snapshot limits for the {vmw-first} Container Storage Interface (CSI) driver without enable the Tech Preview feature gate because of a bug with the Cluster Storage Operator. With this release, you can now change the snapshot limits as expected. For more information about changing the snapshot limits, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.html#vsphere-change-max-snapshot_persistent-storage-csi-vsphere[Changing the maximum number of snapshots for vSphere] (link:https://issues.redhat.com/browse/OCPBUGS-36969[*OCPBUGS-36969*])
```
Maybe better.